content/building-blocks/collect-data/webscraping-apis/scrape-dynamic-websites.md

```diff
@@ -54,12 +54,13 @@ Running the code snippet below starts a new Google Chrome browser (`driver`) and
 {{% codeblock %}}
 ```Python
 import selenium.webdriver
+from selenium.webdriver.common.by import By
```

Comment:
That is correct Cas! Selenium has had an update somewhere in June. Since the updates it is no longer possible to use the old way. More about locating elements in the new fashion can be found [here](https://selenium-python.readthedocs.io/locating-elements.html)

Reply:
A I see. Thanks for pointing this out!